lib/matplotlib/axes/_axes.py

```diff
@@ -632,22 +632,21 @@ def annotate(self, *args, **kwargs):
         s : string
             label
 
-        xy : (x, y)
-            position of element to annotate
+        xy : (x, y) , default: "data"
```

Comment:
This does not default to `'data'`.

I would add something to the body of this parameter like 'see `xycoords` to control what coordinate system this value is interpreted in'  and something similar for `xytext`.